### PR TITLE
Mutation testing

### PR DESCRIPTION
**Pull request by Stearnzy**
### Description
Adds tests for mutations

### Testing
Mutation testing for GraphQL

### Changes include
- [ ] Bugfix (non-breaking change that solves an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

### Checklist
- [x] I have written tests for this code (happy & sad)
- [ ] I have updated the Readme
- [x] I ran full test suite - all tests passing 

### Other comments
2 Users are created and destroyed with each run of the tests, so it throws off the user id in the test DB, however the only tests that actually call on id are fixed 1-3.

### CODE REVIEW

**Review by ShaunDaneJames — APPROVED** [review]

**Review by Shawntru — APPROVED** [review]
Testing looks solid!

File: code/api/src/modules/user/tests/userMutations.test.js [hunk 1]
```diff
@@ -14,4 +16,54 @@ describe('user mutations', () => {
       })
     )
   })
+
+  beforeAll(async () => {
```
Comment (smcunning) [inline code/api/src/modules/user/tests/userMutations.test.js:20]: @Stearnzy you did a great job figuring out this syntax! Especially this `models.User.create` - with missing Rails magic, having to call on the `models` import definitely didn't come intuitively to me!

**Review by alyssajoyner — APPROVED** [review]

File: code/api/src/modules/user/tests/userMutations.test.js (hunk 1 above)
Comment (Stearnzy) [reply code/api/src/modules/user/tests/userMutations.test.js:20]: Thanks!! Took some looking around to find the `models` syntax, but I'm glad it worked too!

File: code/api/src/modules/user/tests/userMutations.test.js [hunk 2]
```diff
@@ -14,4 +16,54 @@ describe('user mutations', () => {
...
+      where: {
+        email: 'stylesssteve@aww.com'
+      }
+    });
+  })
+
+  it('Can successfully update a user style from null', async (done) => {
+    const response = await request(server)
+    .post('/')
+    .send( {query: 'mutation {userStyleUpdate(email: "stylesssteve@aww.com", style: "Bohemian") {email, style}}'} )
```
Comment (smcunning) [inline code/api/src/modules/user/tests/userMutations.test.js:53]: Another really hard won little line of code! Interesting how GraphQL takes in queries in string format.

**Review by smcunning — APPROVED** [review]

**Comment by smcunning** [comment]
I know we were talking about sad path testing for this feature on the backend but weren't able to fully find a good use case for sad paths.

**Pull request merged**